src/test/obj_ulog_size/obj_ulog_size.c

```diff
@@ -493,6 +493,184 @@ do_tx_max_alloc_tx_publish(PMEMobjpool *pop)
 	}
 }
 
+/*
+ * do_log_intent_max_size_limits -- test upper and lower limit of the
+ * pmemobj_tx_log_intent_max_size function argument
+ */
+static void
+do_log_intent_max_size_limits(void)
+{
+	UT_OUT("do_log_intent_max_size_limits");
+
+	size_t size = 0;
+
+	/* lower limit */
+	size = pmemobj_tx_log_intent_max_size(0);
+	UT_ASSERT(size > 0 && size != SIZE_MAX);
+
+	/* upper limit - simplified */
+	size = pmemobj_tx_log_intent_max_size(
+		SIZE_MAX / TX_INTENT_LOG_ENTRY_OVERHEAD);
+	UT_ASSERT(size == SIZE_MAX && errno == EOVERFLOW);
```

Comment:
please use UT_ASSERTeq whenever possible - when the condition is not true it prints the value of both sides